### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mariiamad**
- [DEMO LINK](https://mariiamad.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://mariiamad.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,7 +8,43 @@
...
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Laptops&Computers" class="nav__link data-qa">Laptops & Computers</a>
```
Comment (maxim2310) [inline src/index.html:32]:
```suggestion
            <a href="#Laptops&Computers" class="nav__link" data-qa="hover">Laptops & Computers</a>
```
image

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+  display: flex;
+  justify-content: flex-end;
+  background-color: white;
+  align-items: center;
+  padding: 0 50px;
+  cursor: pointer;
+}
+
+.logo {
+  margin-right: auto;
```
Comment (maxim2310) [inline src/style.css:26]: you just need to set justify-content: space-between; for .header instead this
image

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+.nav__link {
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-left: 20px;
+  text-align: center;
+  position: relative;
+  z-index: 50;
```
Comment (maxim2310) [inline src/style.css:52]:
```suggestion
  position: relative;

```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,92 @@
...
+}
+
+.nav__link::before {
+  position: absolute;
+  content: " ";
+  top: -20px;
+  right: -20px;
+  left: -20px;
+  bottom: -20px;
+  z-index: 40;
```
Comment (maxim2310) [inline src/style.css:62]: dont use so huge z-index its can cause for unexpected behavior,
and why do you need this before?

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,92 @@
...
+}
+
+.nav__link:active {
+  outline: 2px solid #00acdc;
```
Comment (maxim2310) [inline src/style.css:66]: #00acdc this color must be variable

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,92 @@
...
+}
+
+.nav__link:hover {
+  color: blue;
```
Comment (maxim2310) [inline src/style.css:70]: use #00acdc this color like variable

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,92 @@
...
+.is-active::after {
+  position: absolute;
+  content: " ";
+  left: 0;
+  right: 0;
+  bottom: -21px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  height: 4px;
+  flex-shrink: 0;
```
Comment (maxim2310) [inline src/style.css:91]:
```suggestion
```

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good job! Let's improve your code a bit)
And try to restart your tests and pass them all

**Comment by mariiamad** [comment]
Thanks for checking! I improved my code)
I restarted my tests and  passed tham all.

**Pull request closed without merging**